### DISCUSSION TITLE
Challenge not getting solved for Indeed

### DISCUSSION THREAD

**Discussion by AakashDev-Draup** [FlareSolverr/FlareSolverr#1074] [Q&A]
curl -L -X POST 'http://localhost:8191/v1' \
-H 'Content-Type: application/json' \
--data-raw '{
  "cmd": "request.get",
  "url":"https://www.indeed.com/jobs?q=simplot&l=&vjk=06d5953504fda754",
  "maxTimeout": 60000
}'

**Comment by ilike2burnthing** [comment]
https://github.com/FlareSolverr/FlareSolverr/issues/1036
